### PR TITLE
Restructure setup

### PR DESCRIPTION
**Pull request by bliemli**
This change primarily enables the display of the login instructions for baloise.

### CODE REVIEW

File: content/en/setup/_index.md [hunk 1]
```diff
@@ -9,21 +9,16 @@ menu:
 
 ## Setup instructions
 
-This training depends on the installation of `{{% param cliToolName %}}`.
+This training depends on `{{% param cliToolName %}}`, the {{% param distroName %}} command-line interfac command-line interface.
```
Comment (chrira) [inline content/en/setup/_index.md:12]: typo ?

File: content/en/setup/web-terminal.md [hunk 2]
```diff
@@ -5,18 +5,20 @@ type: docs
...
 
-### {{% task %}} Login on the web console
+## {{% task %}} Login on the web console
 
 First of all, open your browser.
 Then, log in on OpenShift's web console using the URL and credentials provided by your trainer.
 
 
-### {{% task %}} Initialize terminal
+## {{% task %}} Initialize terminal
```
Comment (chrira) [inline content/en/setup/web-terminal.md:21]: e.g. add warning to not choose a project and create a new one in the drop-down to prevent common errors.

**Review by chrira — CHANGES_REQUESTED** [review]

File: content/en/setup/_index.md (hunk 1 above)
Comment (bliemli) [reply content/en/setup/_index.md]: Fixed, thanks!

File: content/en/setup/web-terminal.md (hunk 2 above)
Comment (bliemli) [reply content/en/setup/web-terminal.md:21]: Good idea, I added a warning.

**Comment by bliemli** [comment]
Thanks for your review!

**Pull request merged**